packages/agoric-cli/src/deploy.js

```diff
@@ -245,7 +248,11 @@ export default async function deployMain(progname, rawArgs, powers, opts) {
         let lastUpdateCount;
         let stillLoading = [...need].sort();
         progressDot = 'o';
-        while (stillLoading.length) {
+        const untilNotLoading = asyncGenerate(() => ({
+          done: !stillLoading.length,
+          value: stillLoading,
```

Comment:
Curious you chose to provide `stillLoading` as the `value` even though no one ever looks. No change suggested.